.github/pull_request_template.md

```diff
@@ -1,8 +1,11 @@
 ## Description
 
-<!--Why this PR? What is changed? What is the effect? etc.-->
+Update pull_request_template to...
 
-If you haven't already, read the full [contribution guide](../CONTRIBUTING.md). The guide may have changed since the last time you read it, so please double-check. Once you are done and ready to submit your PR, edit the PR description and run through the relevant checklist below.
+- Fix Broken Link
+- Include updating module's `README.md` in precommit checklist
```

Comment:
Are those temporary notes? I think we need to restore it to the `<!-- ... -->` comment.

Reply:
lol, i put my PR notes in the wrong place!